src/client/Microsoft.Identity.Client/json/Serialization/DefaultContractResolver.cs

```diff
@@ -1379,7 +1379,7 @@ protected virtual IValueProvider CreateMemberValueProvider(MemberInfo member)
             // warning - this method use to cause errors with Intellitrace. Retest in VS Ultimate after changes
             IValueProvider valueProvider;
             // UNITY flag based on https://github.com/jilleJr/Newtonsoft.Json-for-Unity
-#if !(PORTABLE40 || PORTABLE || DOTNET || ANDROID || iOS || MAC || UNITY)
+#if !(PORTABLE40 || PORTABLE || DOTNET || ANDROID || iOS || MAC || UNITY || UAP10_0_17763)
```

Comment:
Yes, Updating to NetStandard 2 and Uap 10.0.17763 caused some weird errors that needed to be fixed with things like this.
Mostly because the version of the JSON serializer is old and was created during before NetStandard 2